orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -1440,6 +1440,50 @@ Example 4:
           "be false.",
           "T");
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(PassThrough)
+      .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc("Passes input to output as is with a given signal.")
+      .Input(
+          0,
+          "control_signal",
```

Comment:
What if there are two or more control signals? I feel PassThrough should just accepts `[X1, X2, ..., Xn]` and produce `[X1_alias, X2_alias, ..., Xn_alias]`. That is, we probably can merge `control_signal` into `input_tensor`. To avoid generating unnecessary outputs, all outputs should be `optional`, so if an output is not specified, we don't generate it.

Reply:
yea that's more generic. I will change it